src/metabase/pulse/render/body.clj

```diff
@@ -420,6 +482,105 @@
       [:img {:style (style/style {:display :block :width :100%})
              :src   (:image-src image-bundle)}]]}))
 
+(def default-y-pos
+  "Default positions of the y-axes of multiple and combo graphs.
+  You kind of hope there's only two but here's for the eventuality"
+  (conj (repeat "right")
+        "left"))
+
+(defn- join-series
+  [names colors types row-seqs y-axis-positions]
+  (let [joined (map vector names colors types row-seqs y-axis-positions)]
```

Comment:
Not sure what the constraints are but i get super nervous at `(map vector ..)` over so many collections. Do we know for sure these will all have the same number of elements?

Reply:
I whacked the possibly problematic one (the combos), the multis are guaranteed to have same cardinality because that's the cardinality of cards in the multi